Dockerfile

```diff
@@ -191,12 +191,19 @@ RUN xx-verify /out/usr/bin/protoc-gen-jsonschema
 
 FROM alpine:${ALPINE_IMAGE_VERSION} as grpc_web
 RUN apk add --no-cache \
+        autoconf \ 
+        automake \
         build-base \
-        curl \
-        protobuf-dev
-RUN mkdir -p /grpc-web
+        git \
+        libtool
 ARG GRPC_WEB_VERSION
-RUN curl -sSL https://api.github.com/repos/grpc/grpc-web/tarball/${GRPC_WEB_VERSION} | tar xz --strip 1 -C /grpc-web
+RUN git config --global http.version HTTP/1.1 && \
```

Comment:
Is there a reason we need this? Could you add a comment?

Reply:
Yep, this is the reason CI is failing. For some reason it works local, but not in GitHub Actions. I'll try to find a way to do this with the tarball instead of git I think, will ping you when CI is passing.

Reply:
If you're just after the source and not a release artifact, `https://github.com/grpc/grpc-web/archive/${GRPC_WEB_VERSION}.tar.gz` would get it and is faster than using git, btw.
I am not sure what would happen with submodules - guessing it would not fetch them

Using GitHub API (api.github.com) is tricky and usually requires a token pretty quickly.

just a `git clone --recursive` with a tag should work though